web/src/main/java/org/cbioportal/web/parameter/StudyViewFilter.java

```diff
@@ -100,6 +101,14 @@ public void setCnaGenes(List<CopyNumberGeneFilter> cnaGenes) {
         this.cnaGenes = cnaGenes;
     }
 
+    public List<StructuralVariantGeneFilter> getSVGenes() {
```

Comment:
Shouldn't we remove the set/getFusionGenes methods?

Reply:
agree, at certain time. We cannot backfill structure variants from Fusions easily, so we need to give users more time to make transaction from fusions to structure variants.